SB2018/2018-01-12.md

```diff
@@ -0,0 +1,34 @@
+---
+to: All teams
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+Some important things to announce so keep reading:
+
+## Mentoring
+
+We're going to try and provide mentoring support for your team!
+
+If you're interested in having a mentor come and help you, **reply to this email with the times your teams meet**, and we'll see if we can assign someone to you!
+
+## Tech Days
+
+We're happy to announce some tech days!
+
+They'll be at the University of Southampton Electronics and Computer Science department in the £4 million electronics labs.
```

Comment:
I'm not sure we need to mention the price in this email, it's rather irrelevant and arguably distracting. Saying something (brief) about how they're a great place to have a tech-day would be a good alternative if we want to pad this sentence.

How about "department in the recently refurbished electronics labs"?